src/platform/errors/index.ts

```diff
@@ -13,9 +13,15 @@ export function populateTelemetryWithErrorInfo(props: Partial<TelemetryErrorProp
     props.failed = true;
     // Don't blow away what we already have.
     props.failureCategory = props.failureCategory || getErrorCategory(error);
-    if (props.failureCategory === 'unknown' && isErrorType(error, FetchError)) {
+
+    // In browsers, FetchError is undefined and the error we receive is a `TypeError` with message `Failed to fetch`.
+    const isBrowserFetchError = !FetchError && error?.name === 'TypeError' && error?.message === 'Failed to fetch';
+    const errorType = isBrowserFetchError ? 'TypeError' : FetchError;
+
+    if (props.failureCategory === 'unknown' && isErrorType(error, errorType)) {
```

Comment:
I don't think we should override isErrorType to support just one edge case.
I'd prefer to special case this and just check the types/strings in here for node-fetch.
Basically the second argument in `isErrorType` will almost never be used in other places, if we do find other places then we can make it more generic as you have, else I think its just going to complicate it by supporting a string.